help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md

```diff
@@ -0,0 +1,213 @@
+---
+keywords: email;Email;e-mail;email destinations;salesforce;salesforce destination
+title: Salesforce Marketing Cloud (ExactTarget) connection
+description: The Salesforce Marketing Cloud (ExactTarget) destination allows you to export your account data and activate it within Salesforce Marketing Cloud (ExactTarget) for your business needs.attribute set
+---
+
+# [!DNL Salesforce Marketing Cloud (ExactTarget)] connection
+
+## Overview {#overview}
+
+[[!DNL Salesforce Marketing Cloud (ExactTarget)]](https://www.salesforce.com/products/marketing-cloud/overview/) is a digital marketing suite formerly known as ExactTarget that allows you to build and customize journeys for visitors and customers to personalize their experience.
```

Comment:
[[!DNL Salesforce Marketing Cloud]](https://www.salesforce.com/products/marketing-cloud/overview/) (formerly known as ExactTarget) is [...]